### PR TITLE
test: arrange examples/ and move performance under tests/

### PR DESCRIPTION
**Pull request by pxlxingliang**

### CODE REVIEW

**Review by hongriTianqi — APPROVED** [review]

**Review by caic99 — CHANGES_REQUESTED** [review]
Would you change this line to ensure the running of performance test:

https://github.com/deepmodeling/abacus-develop/blob/509ed5dfa466170b66c6f030d2041ed7f47add5e/.github/workflows/performance.yml#L54

**Comment by pxlxingliang** [comment]
> Would you change this line to ensure the running of performance test:
> 
> https://github.com/deepmodeling/abacus-develop/blob/509ed5dfa466170b66c6f030d2041ed7f47add5e/.github/workflows/performance.yml#L54

ok

File: examples/README [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/*******************************************************************************/
+/
+/ This is the examples of ABACUS program.
```
Comment (caic99) [inline examples/README]:
```suggestion
/ These are the examples of ABACUS program.
```

**Review by caic99 — CHANGES_REQUESTED** [review]

File: examples/scf/lcao_Si2/README [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************/
 /
-/ This is the examples of ABACUS program.
+/ These are the examples of ABACUS program.
 /
 /*******************************************************************************/
```
Comment (caic99) [inline examples/scf/lcao_Si2/README:5]: I suggest removing this part in test cases, only preserved in examples/Readme.

Comment (pxlxingliang) [reply examples/scf/lcao_Si2/README:5]: I think this is just a title, a unified format

Comment (pxlxingliang) [reply examples/scf/lcao_Si2/README:5]: I think it is ok to keep it.

**Review by caic99 — APPROVED** [review]

**Pull request merged**